modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -196,36 +228,16 @@ With the release of the {product-short} 1.2.1 Helm chart, this is fixed.
 //link:{LinkRHIDPIssue}RHIDP-2931[RHIDP-2931]
 --
 
-== Fixed issues in {product} 1.2.2
-
-This section lists fixed issues with {product} 1.2.2:
-
-Dynamic plugins fail to load if the file name in ConfigMap is invalid::
-+
---
-Previously, an invalid dynamic plugin file name in ConfigMap prevented the creation of a Backstage CR. With this release, the Backstage CR can be created even with an invalid plugin file name, and a corresponding log message will be displayed if the RHDH build fails.
---
-
-Plugin ID is missing in the permission table in RBAC Administration UI::
-+
---
-Previously, the plugin ID was missing from the permission table in the *Administration* -> *RBAC*. With this release, updates to both the RBAC backend and frontend plugins now ensure the plugin ID is displayed in the table.
---
-
-Deprecation warning about `backend.auth.keys` in application log::
-+
---
-Previously, RHDH build logs showed the following deprecation warning:
+== Fixed security issues
```

Comment:
Found a cosmetic issue :)

This line should be changed to 
`== Fixed security issues in {product} 1.2.2`
and the next line (`=== Fixed security issues in {product} 1.2.2`) should be removed.

Reply:
@josephca Not necessarily, as we separate the issues by z-stream release and list them in reverse chronological order - see section _6. Fixed issues in Red Hat Developer Hub 1.2 and 1.2.2_

However, section **_6.4. Fixed security issues in Red Hat Developer Hub 1.2_** should be a sub-heading of **_6.3. Fixed security issues_**

Like so:
_6.3. Fixed security issues in Red Hat Developer Hub 1.2 and 1.2.2
6.3.1. Fixed security issues in Red Hat Developer Hub 1.2.2
6.3.2 Fixed security issues in Red Hat Developer Hub 1.2_